pageobjects.py

```diff
@@ -14,8 +12,7 @@ def get_element(self, xpath):
 
     def click_with_js(self, xpath):
         btn_plan = self.get_element(xpath)
-        self.driver.execute_script("arguments[0].click();", btn_plan)\
-
+        self.driver.execute_script("arguments[0].click();", btn_plan)
```

Comment:
just curious, what is the other way to `click`?

Reply:
this one is actually dirty hack , normally you need to call https://www.selenium.dev/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webelement.html#selenium.webdriver.remote.webelement.WebElement.click
but the problem that we have several cases where element which we need to click is overlapped by another element in DOM. So native click is end up with error "Can not click on overlapped element" ( don't remember error text exactly ). Maybe there is better way to handle this but I use this hack which I discovered long time ago :)